bears/php/PHPCodeBeautifierBear.py

```diff
@@ -0,0 +1,177 @@
+            Checks that ``PHP`` keywords are lowercase.
+        :param force_lower_case_constant:
+            Checks that all uses of ``true``, ``false`` and ``null`` are
+            lowercase.
+        :param blank_line_after_namespace_declaration:
+            Ensures that there is a blank line after a namespace declaration.
+        :param check_use_blocks:
+            Ensures that there is one blank line after a ``use`` block,
+            that there is only one use block per line, and that all ``use``
+            declaration are done after namespaces declaration.
+        :param check_class_declaration:
+            Ensures that ``extends`` and ``implements`` keywords are declared
+            on the same line as the class name, that the opening brace for a
+            class is on the next line, and that the closing brace for a class
+            is on the next line after the body. Allows splitting implements
+            list accross multiple lines.
+        :param check_property_declaration:
+            Ensures that visibility is declared on all properties, that the
+            ``var`` keyword is not used to declare a property, that there is
+            not more that one property declared on a line, that properties are
+            not prefixed with an underscore.
+        :param force_scope_modifier_on_method:
+            Verifies that class methods have scope modifiers.
+        :param function_declaration_argument_spacing:
+            Number of spaces between arguments in function declaration.
+        :param allow_multiline_function_declaration:
+            Allows argument lists to be split accross multiple lines correctly
+            indented.
+        """
+        rules_map = {'Generic.WhiteSpace.DisallowTabIndent':
+                     use_spaces,
+                     'Generic.Formatting.DisallowMultipleStatements':
+                     not allow_multiple_statements_per_line,
+                     'Generic.PHP.LowerCaseKeyword':
+                     force_lower_case_keywords,
+                     'Generic.PHP.LowerCaseConstant':
+                     force_lower_case_constants,
+                     'PSR2.Namespaces.UseDeclaration':
+                     check_use_blocks,
+                     'PSR2.Namespaces.NamespaceDeclaration':
+                     blank_line_after_namespace_declaration,
+                     'PSR2.Classes.ClassDeclaration':
+                     check_class_declaration,
+                     'PSR2.Classes.PropertyDeclaration':
+                     check_property_declaration,
+                     'Squiz.Scope.MethodScope':
+                     force_scope_modifier_on_method,
+                     'Squiz.Functions.MultiLineFunctionDeclaration':
+                     allow_multiline_function_declaration}
+        rules = ''
+        for k, v in rules_map.items():
+            rules += '<rule ref="{}"/>\n'.format(k) if v else ''
+        configs = '''<?xml version="1.0"?>
```

Comment:
hmm I think it  might be better if we move this to a file

Reply:
that is whole `generate_config` in another file?

Reply:
no just configs, put in a file like `psr-1.xml` and read it from there and then do the `.format`